### PR TITLE
support 'exclude from purge' feature from doctrine/data-fixtures

### PR DESCRIPTION
**Pull request by patrickmyh**
as per issue #289
new pull request, since I messed up the history of the previous branch ;(

### CODE REVIEW

**Comment by lsmith77** [comment]
thx .. there seems to be a test failure:

```
1) Liip\FunctionalTestBundle\Tests\Test\WebTestCaseConfigMysqlTest::testLoadFixturesAndExcludeFromPurge
Failed asserting that 0 is identical to 2.
/home/travis/build/liip/LiipFunctionalTestBundle/Tests/Test/WebTestCaseConfigMysqlTest.php:156
```

**Comment by patrickmyh** [comment]
cannot reproduce this failing test.
am on commit dc7eb1b2fbb2f2c508c121f7ee0bb4ed1a96128b, reinstalled the vendors again. everything works.

**Comment by lsmith77** [comment]
it seems to be related to your PR because master tests run through, I just retriggered them.

But these two tests run fails:
https://travis-ci.org/liip/LiipFunctionalTestBundle/jobs/188289852
https://travis-ci.org/liip/LiipFunctionalTestBundle/jobs/188289844

I suspect its related to version lower than 5.6 https://travis-ci.org/liip/LiipFunctionalTestBundle/builds/188289843

**Comment by patrickmyh** [comment]
duh! it appears, that composer's depencies are the problem here.
for PHP_VERSION 5.4 and 5.5, an older version of the doctrine/data-fixtures vendor is retrieved, one that is below version 1.2.2 and which thus doesn't support the feature the pull request was all about.

nice, though, that the tests showed this.
do you know why the old version of doctrine/data-fixtures is used for said php versions? or do these still need to be supported?

**Comment by alexislefebvre** [comment]
PHP 5.4 and 5.5 have reached end of life, they are unsupported now: http://php.net/supported-versions.php

@lsmith77 Can we remove 5.4 and 5.5 from the test matrix?

**Comment by lsmith77** [comment]
hmm yeah I guess if its additional hassle lets drop them.

**Comment by lsmith77** [comment]
@patrickmyh can you adjust the build matrix accordingly?

**Comment by alexislefebvre** [comment]
@patrickmyh Can you please document the usage of this new method in the README?

File: .travis.yml [hunk 1]
```diff
@@ -18,8 +17,6 @@ services:
 
 matrix:
   include:
-    - php: 5.4
-      env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.6
       env: SYMFONY_VERSION="~2.3.27 symfony/console:2.5.*"
```
Comment (lsmith77) [inline .travis.yml]: I would like to keep a ``prefer-lowest`` test, can you change this line to ``COMPOSER_FLAGS="--prefer-lowest"`` ?

**Pull request merged**